core/src/main/scala/caliban/interop/circe/circe.scala

```diff
@@ -19,3 +26,13 @@ private[caliban] trait IsCirceDecoder[F[_]]
 private[caliban] object IsCirceDecoder {
   implicit val isCirceDecoder: IsCirceDecoder[Decoder] = null
 }
+
+object json {
+  implicit val jsonSchema: Schema[Any, Json] = new Schema[Any, Json] {
+    override def toType(isInput: Boolean): __Type = makeScalar("Json")
+    override def resolve(value: Json): Step[Any] =
+      QueryStep(ZQuery.fromEffect(ZIO.fromEither(implicitly[Decoder[ResponseValue]].decodeJson(value))).map(PureStep))
```

Comment:
Use `Decoder[ResponseValue]` instead of `implicitly[Decoder[ResponseValue]]` all cats based libs have this summoner pattern.

Reply:
Good point!